website/blog/2024-08-08-dso-toolkit-63.0.0.mdx

```diff
@@ -0,0 +1,11 @@
+---
+authors: [iterox]
+---
+
+# DSO Toolkit v63.0.0 🎸
+
+In deze breaking release is het legacy HTML/CSS Component Panel verwijderd.
+
+## Panel (legacy)
+
+In deze breaking release hebben we het legacy HTML/CSS Component Panel verwijderd. Dit component is volgens de oude deprecation policy al eens deprecated.
```

Comment:
Ik wil de afnemers geruststellen en aangeven dat het een _formele_ BREAKING release is:

```suggestion
In deze breaking release hebben we het legacy HTML/CSS Component Panel verwijderd. Dit component is volgens de oude deprecation policy al eens deprecated en zou in principe geen impact voor afnemers betekenen.

Het gaat om:

<!-- ```html -->
<div class="panel">
<!-- ``` -->

```